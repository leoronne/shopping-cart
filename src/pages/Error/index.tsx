import React  from 'react';

import { Container, Title } from './styles';
import error from '~/assets/img/error.png';

const Error: React.FC = () => {
  return (
    <Container>
    <img src={error} alt="Error" className="error-bg"/>
      <Title>
          Oops...
      </Title>
      <p>Sorry, we had a problem loading our resources.</p>
      <p>Please, try again later!</p>
    </Container>
  );
};

export default Error;
