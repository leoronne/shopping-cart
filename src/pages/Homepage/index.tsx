import React from 'react';

import { ProductsContainer, ShoppingCartContainer } from '../../components';

import { Container, GridContainer } from './styles';

const Homepage: React.FC = () => {
  return (
    <Container data-testid="product-cointainer">
      <GridContainer>
        <ProductsContainer />
        <ShoppingCartContainer />
      </GridContainer>
    </Container>
  );
};

export default Homepage;
