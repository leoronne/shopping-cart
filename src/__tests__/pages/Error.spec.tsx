import React from 'react';
import { render } from '@testing-library/react';

import { Error } from '../../pages';

describe('Error', () => {
  it('should render', async () => {
    const { asFragment } = render(<Error />);

    const html = asFragment();

    expect(html).toMatchSnapshot();
  });
});
