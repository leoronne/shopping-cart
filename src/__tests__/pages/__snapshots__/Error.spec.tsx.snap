// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render 1`] = `
<DocumentFragment>
  <div
    class="sc-iwyYcG eZCCGl"
    data-testid="error-cointainer"
  >
    <img
      alt="Error"
      class="error-bg"
      src="error.png"
    />
    <div
      class="sc-cxFLnm igzjKP"
    >
      Oops...
    </div>
    <p>
      Sorry, we had a problem loading our resources.
    </p>
    <p>
      Please, try again later!
    </p>
  </div>
</DocumentFragment>
`;
